src/PdfSharp/Pdf/PdfDocument.cs

```diff
@@ -725,8 +725,7 @@ public PdfOutlineCollection Outlines
         /// </summary>
         public PdfAcroForm AcroForm
         {
-            get { return Catalog.AcroForm; }
-            set { Catalog.AcroForm = value; }
```

Comment:
I think he added it purely for his test console project. If we find a need to add it back we can.